src/api/spec/policies/comment_policy_spec.rb

```diff
@@ -72,6 +72,16 @@
       it { is_expected.to permit(user, comment_on_request) }
       it { is_expected.not_to permit(other_user, comment_on_request) }
     end
+
+    context 'with a comment of a Report' do
```

Comment:
```suggestion
    context 'with a comment for a report' do
```

Reply:
A search of "comment of a" returns 4 ocurrences: https://github.com/search?q=%22comment+of+a%22+repo%3Aopensuse%2Fopen-build-service&type=code&ref=advsearch

A search of "comment for a" returns onely 1: https://github.com/search?q=%22comment+for+a%22+repo%3Aopensuse%2Fopen-build-service&type=code&ref=advsearch

We should agree on some consensus here. :thinking:

Reply:
Well, If I'm not wrong, "comment of a report" is the comment coming from a report, while "comment for a report" is the comment about a report, so regardless of what we already have, the right form is the second one.

Reply:
It could also be "comment on a report"